src/lib/coda_base/pending_coinbase.ml

```diff
@@ -367,14 +367,7 @@ module T = struct
      versions yields a version of the result
    *)
   module Merkle_tree = struct
-    module Stable = struct
-      module V1 =
-        Sparse_ledger_lib.Sparse_ledger.Make
-          (Hash.Stable.V1)
-          (Stack_id.Stable.V1)
-          (Stack.Stable.V1)
-      module Latest = V1
-    end
+    include Sparse_ledger_lib.Sparse_ledger.Make (Hash) (Stack_id) (Stack)
```

Comment:
Why get rid of the versioning that depends on the `Make` functor inputs?

This `Merkle_tree` is mentioned below in a `bin_io` type `t`, which will need to versioned itself. Such a versioned `t` must contain only versioned types.